kernel/interrupts/src/aarch64/mod.rs

```diff
@@ -119,8 +119,16 @@ fn set_vbar_el1() {
 }
 
 /// Sets `VBAR_EL1` to the start of the exception vector
+/// and enables timer interrupts
 pub fn init_ap() {
     set_vbar_el1();
+
+    // Enable the CPU-local timer
+    let mut gic = GIC.lock();
+    let gic = gic.as_mut().expect("GIC is uninitialized");
```

Comment:
this is a bug, right? we should clarify a bit
```suggestion
    let gic = gic.as_mut().expect("BUG: init_ap(): GIC was uninitialized");
```